RedHatInsights/insights-advisor-frontend
# fix(advisor-3041): Update advisor comp to fix broken links

# Description
ticket advisor-3041

how to test:
launch advisor against production
use insights-qa account
check the system https://console.redhat.com/insights/advisor/systems/3cce4a57-acaf-4974-a0b1-84961354a5de
if it's not exist - request it from rodrigo antunes


![image](https://github.com/RedHatInsights/insights-advisor-frontend/assets/62722417/4b3e7b5d-8f28-4c2f-8936-9bcd643b94f2)
